packages/dds/tree/src/test/shared-tree/schematizingTreeView.spec.ts

```diff
@@ -50,25 +52,39 @@ describe("SchematizingSimpleTreeView", () => {
 		const checkout = checkoutWithContent(emptyContent);
 		const view = new SchematizingSimpleTreeView(checkout, config, new MockNodeKeyManager());
 
-		const error: SchematizeError | undefined = view.error;
-		assert(error instanceof SchematizeError);
-		assert(error.canInitialize === true);
-		assert(error.canUpgrade === true);
-		assert(error.updateType === UpdateType.Initialize);
+		assert.throws(
+			() => view.root,
+			(e) => e instanceof UsageError,
```

Comment:
we have a validateUsageError helper you might want to use here. Makes it easy to check the error is relevant (ex: check it contains \compatibility\ or something like that.